send-transaction-service/src/send_transaction_service.rs

```diff
@@ -311,14 +302,26 @@ impl SendTransactionService {
         result
     }
 
-    fn send_transaction(
-        send_socket: &UdpSocket,
-        tpu_address: &SocketAddr,
-        wire_transaction: &[u8],
-    ) {
-        if let Err(err) = send_socket.send_to(wire_transaction, tpu_address) {
+    fn send_transaction(tpu_address: &SocketAddr, wire_transaction: &[u8]) {
+        let mut measure = Measure::start("send_transaction_service-us");
+        let connection = connection_cache::get_connection(tpu_address);
+
+        if let Err(err) = connection.send_wire_transaction(wire_transaction) {
             warn!("Failed to send transaction to {}: {:?}", tpu_address, err);
         }
+        measure.stop();
+        inc_new_counter_info!(
+            "send_transaction_service-us",
+            measure.as_us() as usize,
+            1000,
+            1000
+        );
+        inc_new_counter_info!(
+            "send_transaction_service-bytes",
+            wire_transaction.len(),
+            1000,
+            1000
+        );
```

Comment:
thanks! I think we can omit the data, since that will be the same regardless of the protocol

Reply:
Good to know. I will remove